From e35033902398d4e433e7694c11a17db7f44a7243 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Tue, 4 Apr 2023 22:01:28 +0300
Subject: [PATCH 2/5] base: Don't use updatable apexes

This is useless for us but consumes some boot time.

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 common/base/device.mk | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/common/base/device.mk b/common/base/device.mk
index d69a8ef..98b9788 100644
--- a/common/base/device.mk
+++ b/common/base/device.mk
@@ -18,9 +18,6 @@ ifneq ($(filter $(TARGET_ARCH),arm64 x86_64),)
     $(call inherit-product, $(SRC_TARGET_DIR)/product/core_64_bit.mk)
 endif
 
-# Enable updating of APEXes
-$(call inherit-product, $(SRC_TARGET_DIR)/product/updatable_apex.mk)
-
 # Enable userspace reboot
 $(call inherit-product, $(SRC_TARGET_DIR)/product/userspace_reboot.mk)
 
-- 
2.37.2

